ota-plus-web/app/reactapp/src/components/groups/CreateModal.jsx

```diff
@@ -167,14 +167,26 @@ class CreateModal extends Component {
     const currentStep = this.steps[this.currentStepId];
     const step = (
       <span>
-        {<currentStep.class wizardData={this.wizardData} onStep1DataSelect={this.onStep1DataSelect} onStep2DataSelect={this.onStep2DataSelect} />}
-        <div className='body-actions'>
+        {<currentStep.class groupType={this.wizardData.groupType} onStep1DataSelect={this.onStep1DataSelect} onStep2DataSelect={this.onStep2DataSelect} />}
+        <div className="body-actions">
           {this.isLastStep() ? (
-            <Button htmlType='button' className='btn-primary pull-right' id='wizard-launch-button' disabled={!currentStep.isFinished} onClick={this.createGroup}>
+            <Button
```

Comment:
Please avoid formatting/prettying/whitespace changes in `feat` and `bugfix` PRs. Better to open separate `refactor` PR.

Reply:
Thank you. According to conversation with Greg we are going to improve source code in little steps in current tickets. Next we will create some Jira tasks to refactor the code.